src/exceptions/GPL-CC-1.0.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <exception licenseId="GPL-CC-1.0" name="GPL Cooperation Commitment 1.0">
+    <crossRefs>
+      <crossRef>https://gplcc.github.io/</crossRef>
+    </crossRefs>
+    <notes>
+      Single exception covering three variants:
+      Individual, Company, and Project.
+    </notes>
+    <titleText>
+      <optional>
+      <p>
+        GPL Cooperation Commitment Version 1.0
+      </p>
+      </optional>
+    </titleText>
+    <p>
+      <optional>Solely for any software for which I personally own
+      copyright that is licensed under a Covered License,</optional>
+      <alt name="before" match="before|Before">before</alt>
+      filing or continuing to prosecute any legal proceeding or claim
+      (other than a Defensive Action) arising from termination of a
+      Covered License,<alt name="subject" match="I|we|.*">we</alt>
+      commit<optional>s</optional>
+      to extend to the person or entity ("you") accused of violating
+      the Covered License the following provisions regarding cure and
+      reinstatement, taken from GPL version 3. As used here, the term
+      "this License" refers to the specific Covered License being enforced.
+    </p>
+    <p>
+      <optional>"</optional>However,
+      if you cease all violation of this License, then your
+      license from a particular copyright holder is reinstated (a)
+      provisionally, unless and until the copyright holder explicitly
+      and finally terminates your license, and (b) permanently, if
+      the copyright holder fails to notify you of the violation by
+      some reasonable means prior to 60 days after the cessation.
+    </p>
+    <p>
+      Moreover, your license from a particular copyright holder is
+      reinstated permanently if the copyright holder notifies you of
+      the violation by some reasonable means, this is the first time you
+      have received notice of violation of this License (for any work)
+      from that copyright holder, and you cure the violation prior to
+      30 days after your receipt of the notice.<optional>"</optional>
+    </p>
+    <p>
+      <alt name="subject" match="I|We|.*">We</alt>
+      intend<optional>s</optional>
+      this Commitment to be irrevocable, and binding and enforceable
+      against <alt name="object" match="me|us|.*">us</alt>
```

Comment:
Same comment as above - Since .* will match any of the words, the other alternatives will be ignored.  I'm thinking we just restrict it to I and We and remove the .*

Reply:
same reply. need COMPANY_NAME